Example/Example/Shared/ConfigurationStub.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ConfigurationStub.swift
+//  Example
+//
+//  Created by Adam Campbell on 12/8/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Afterpay
+import Foundation
+
+var configurationStub = ConfigurationStub()
+
+struct ConfigurationStub {
+
+  var minimumAmount: String? = "5.00" {
+    didSet { updateConfiguration() }
+  }
+
+  var maximumAmount: String = "1000.00" {
+    didSet { updateConfiguration() }
+  }
+
+  var currencyCode: String = "USD" {
```

Comment:
Could this default to `Settings.currencyCode`?

Reply:
With the config override it might be best to ditch that one